docs/api/graphql_queries.md

```diff
@@ -181,6 +181,72 @@ Response:
 }
 ```
 
+### Get Matrix Field Type
+
+To get a Matrix Field Type, use `product` field.
+
+The example below shows Product Content Type that has two fields:
+
+- `name`: `ezstring`
+- `features`: `ezmatrix` with two columns: `name` and `price`
+
+Fields of that type return a list of `ProductFeaturesRow`.
+
+```
+{
+  content {
+    product(contentId: 123) {
+      name
+      features {
```

Comment:
At this point, we must explain that the type returned for a matrix field exposes the columns defined in the definition. The example is valid.